src/libs/KeyboardShortcut/index.js

```diff
@@ -103,8 +103,11 @@ function bindHandlerToKeydownEvent(event) {
 }
 
 // Make sure we don't add multiple listeners
-document.removeEventListener('keydown', bindHandlerToKeydownEvent, {capture: true});
-document.addEventListener('keydown', bindHandlerToKeydownEvent, {capture: true});
+const rootNode = document.getElementById('root');
+rootNode.removeEventListener('keydown', bindHandlerToKeydownEvent);
+rootNode.addEventListener('keydown', bindHandlerToKeydownEvent);
+window.removeEventListener('keydown', bindHandlerToKeydownEvent);
+window.addEventListener('keydown', bindHandlerToKeydownEvent);
```

Comment:
I believe this code is not required. Could you explain why did you add this?

Reply:
I don't recall right now, but found that it was necessary in my testing, which was pretty thorough for the login page.

Reply:
I think the idea is that we establish these event handlers in the bubbling phase so that further down in the DOM we can prevent them from being triggered in the capture phase.

If you have a capture-phase handler at the document root, then there's no way to prevent it from executing first